### PR TITLE
Product Listing page 

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
Added product listing page.

### CODE REVIEW

File: css/product.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
+.main-container{
+    margin: 8vh 1rem 0 1rem;
+    background-color: #eeeeee;
+    position: relative;
+    display:flex;
+    height: 100%;
+    gap: 20px;
```
Comment (pritam-kr) [inline css/product.css:7]: use relative unit

**Review by pritam-kr — APPROVED** [review]
Good to merge

File: css/product.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
+.main-container{
+    margin: 8vh 1rem 0 1rem;
+    background-color: #eeeeee;
```
Comment (pratik-kumar-sahu) [inline css/product.css]: use var for colors

**Review by pratik-kumar-sahu — APPROVED** [review]
Else looks good

**Review by Aazam10 — APPROVED** [review]
Great work.

**Pull request merged**